EIPS/eip-7733.md

```diff
@@ -0,0 +1,60 @@
+---
+eip: 7733
+title: Deactivate EIP-158
+description: Deactivates EIP-158 in order to avoid conflicts in a context in which statelessness and EIP-7702 are active
+author: Guillaume Ballet (@gballet)
+discussions-to: https://ethereum-magicians.org/t/eip-7733-eip-158-deactivation/20445
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-07-02
+requires: 158, 7702
+---
+
+## Abstract
+
+Deactivate [EIP-158](./eip-158.md).
+
+## Motivation
+
+After [EIP-6780](./eip-6780.md), the `SELFDESTRUCT` instruction was neutered, so that contracts could no longer be deleted. Contract deletions can still occur, however, via EIP-158. While [EIP-7702](./eip-7702.md) this can not happen, this latter EIP introduces the possibility for an empty EoA to have state. This is causing some issues when interfacing with verkle, for which state deletion is not possible.
+
+EIP-158 was meant as a temporary measure to combat the "Shanghai attacks". Now that this is attack has been mitigated by other means, it can be deactivated.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+|Name|Value|
+|----|-----|
+|`FORK_TIME`|TBD|
```

Comment:
```suggestion
|`FORK_TIME`|TBD <!-- TODO --> |
```

If you use an HTML-style comment, the linter will make sure you replace it before moving to the Review status.